README.md

```diff
@@ -1,87 +1,159 @@
-1. Usage of Jira ticket numbers in the subject line.
-    - Because the subject line is expensive real estate every character should be most efficiently used to convey meaning to humans.
-    - Jira ticket numbers are not equal to the tickets themselves and thus convey very little information.
-      These ticket numbers should go in message footers where tools can still extract them for automatic linking.
-2. Using non-imperative verb forms (adds, added or adding instead of add) in the subject line.
-    - These other forms convey no more meaning but use extra precious characters.
-3. Referring to review comments that cannot be found anywhere in the commit history itself.
-    - Commit messages should be self-contained.
-      Only mentioning that a commit is created in response to a review comment, without mentioning the reasoning of that comment is clearly not self-contained.
-    - Whenever your workflow permits it, prefer amending the original commit (or using `--fixup`) instead.
-    - If your workflow doesn't permit that, or it seems suboptimal in a given context, describe what, why and how you are changing (as mentioned before).
 
 ## Installation
 
 You can install this package with pip as follows:
 
 ```sh
-pip3 install --user --upgrade commisery
+$ pip3 install --user --upgrade commisery
 ```
 
 ## Usage
 
-You can verify commit messages with the included CLI tool:
+Basic usage instructions:
+
+```
+Usage: cm [OPTIONS] COMMAND [ARGS]...
+
+  Manages conventional commit messages
+
+Options:
+  -c, --config TEXT               Path towards a configuration file
+  -t, --tags TEXT                 Comma-separated list of accepted
+                                  conventional commit tags to allow aside from
+                                  the default "feat" and "fix". If omitted,
+                                  uses the following list: fix, feat, build,
+                                  chore, ci, docs, perf, refactor, revert,
+                                  style, test, improvement
+
+  -l, --max-subject-length INTEGER
+                                  Maximum characters allowed in the subject of
+                                  the commit message
+
+  -d, --disable TEXT              List of commit message rules to disable. Can
+                                  be one of: C001, C002, C003, C004, C005,
+                                  C006, C007, C008, C009, C010, C011, C012,
+                                  C013, C014, C015, C016, C017, C018, C019,
+                                  C020
+
+  -v, --verbosity LVL             Either CRITICAL, ERROR, WARNING, INFO or
+                                  DEBUG
+
+  --help                          Show this message and exit.
+
+Commands:
+  check     Checks whether commit messages adhere to the Convention Commits...
+  commit    Creates a conventional commit
+  overview  Lists the accepted Conventional Commit tags and Rules (incl.
```

Comment:
Missing part of the sentence behind "incl. "?

Reply:
Default behavior of `Click`, full description is visible with `cm overview --help`